real-main/app/models/card/templates.py

```diff
@@ -144,3 +144,19 @@ def __init__(self, user_id, requested_followers_count):
         self.card_id = self.get_card_id(user_id)
         cnt = requested_followers_count
         self.title = f'You have {cnt} pending follow request{"s" if cnt > 1 else ""}'
+
+
+class UserSubscriptionLevelTemplate(CardTemplate):
+    
+    action = 'https://real.app/diamond/'
```

Comment:
```suggestion
    action = 'https://real.app/diamond'
```

This is for consistency with (most) of the other card actions.